SI/Sem.02/Pract.02/Task02.cpp

```diff
@@ -0,0 +1,33 @@
+﻿#include <iostream>
+
+int main() {
+	char inputCharacter;
+
+	std::cin >> inputCharacter;
+
+	if ((inputCharacter >= 'A' && inputCharacter <= 'Z') || (inputCharacter >= 'a' && inputCharacter <= 'z')) {
+		switch (inputCharacter) {
+			case 'A':
+			case 'E':
+			case 'I':
+			case 'O':
+			case 'U':
+			case 'W':
+			case 'Y':
+			case 'a':
+			case 'e':
+			case 'i':
+			case 'o':
+			case 'u':
+			case 'w':
+			case 'y':
+				std::cout << "Entered letter is a vowel.\n";
+				break;
+			default:
+				std::cout << "Entered letter is not a vowel.\n";
+				break;
+		}
+	}
```

Comment:
else cout<<"The symbol is not part of the latin alphabet"

Reply:
Alright. Will do.